### PR TITLE
remove usage of test_owners in test-infra

### PR DESCRIPTION
**Pull request by neolit123**
- don't use test_owners.csv in the issue-creator robot
- remove fetching of k/k's test_owners in update_summaries.sh
- remove test_owners lines from prow/gitattributes/gitattributes_test.go

xref https://github.com/kubernetes/test-infra/issues/14839
https://github.com/kubernetes/kubernetes/pull/84070

/kind cleanup
/priority backlog

### CODE REVIEW

File: config/jobs/kubernetes/test-infra/fejta-bot-periodics.yaml [hunk 1]
```diff
@@ -284,7 +284,6 @@ periodics:
       - --org=kubernetes
       - --project=kubernetes
       - --token-file=/etc/token/bot-github-token
-      - --test-owners-csv=/test_owners.csv
```
Comment (neolit123) [inline config/jobs/kubernetes/test-infra/fejta-bot-periodics.yaml]: AFAIK this should not affect the output of the flaky test reports?
https://github.com/kubernetes/kubernetes/issues/83870

**Comment by cblecker** [comment]
LGTM, but I defer to @fejta on the issue creator bits

**Pull request merged**